spec/factories/openid.rb

```diff
@@ -0,0 +1,10 @@
+# Read about factories at https://github.com/thoughtbot/factory_girl
+
+FactoryGirl.define do
+  factory :openid do
+    provider 'test'
+    email 'test@test.com'
```

Comment:
You could use Faker to generate the factory fields, so you do not always tests it with the same case.

Reply:
Done